tests/ish_report_test.py

```diff
@@ -111,7 +111,7 @@ def test_fm15(self):
     self.assertEqual(weather.report_type, 'FM-15')
     self.assertEqual(weather.elevation, 205)
     self.assertEqual(weather.wind_speed, 1.5)
-    self.assertEqual(weather.wind_speed.get_miles(), 3.3554)
+    self.assertEqual(weather.wind_speed.get_MilesPerHour(), 3.3554)
```

Comment:
Can we change this to snake case to fit in with convention? get_miles_per_hour / get_mph